SETUP.md

```diff
@@ -50,18 +50,11 @@ Next you must clone `manubot/rootstock` and configure its branches and remotes:
 
 ```sh
 # Clone manubot/rootstock
-git clone https://github.com/manubot/rootstock.git $REPO
+git clone --single-branch https://github.com/manubot/rootstock.git $REPO
```

Comment:
the effect of this can be viewed by doing `git branch --all`.

Before --single-branch

```
(base) ~/D/r/rootstock-branchless-setup ❯❯❯ git branch -a
* master
  remotes/origin/HEAD -> origin/master
  remotes/origin/gh-pages
  remotes/origin/master
  remotes/origin/output
```

After --single-branch

```
(base) ~/D/r/rootstock-branchless-setup ❯❯❯ git branch -a
* master
  remotes/origin/HEAD -> origin/master
  remotes/origin/master
```